extension/scripts/features/collapsibleareas/ttCollapsibleAreas.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Collapse Areas",
+		"sidebar",
+		() => settings.pages.sidebar.collapseAreas,
+		null,
+		addCollapseIcon,
+		removeCollapseIcon,
+		{
+			storage: ["settings.pages.sidebar.collapseAreas"],
+		},
+		null
+	);
+
+	async function addCollapseIcon() {
+		await requireSidebar();
+		if (!document.find("[class='header-arrow']") && !mobile) {
+			let areasIElement = document.newElement({ type: "i", class: "icon fas fa-caret-down" });
+			let areasHeader = document.find("h2=Areas");
+			areasHeader.classList.add("tt-title-torn");
+			areasHeader.appendChild(areasIElement);
+			if (filters.containers.collapseAreas) areasHeader.classList.add("collapsed");
+			areasHeader.addEventListener("click", clickListener);
+		}
+	}
+
+	async function removeCollapseIcon() {
+		await requireSidebar();
+		if (!document.find("[class='header-arrow']") && !mobile) {

```

Comment:
If nothing else is done besides the `if`, I'd rather have it return when it's wrong, no need to create an indent for the entire good flow. 

Since this feature is not enabled though due to these things, you can just remove this entirely.